components/collective-page/sections/Contributors.js

```diff
@@ -15,6 +15,9 @@ import SectionTitle from '../SectionTitle';
 
 import ContributorsGridBackgroundSVG from '../../../public/static/images/collective-page/ContributorsGridBackground.svg';
 
+/* reference to the FixedSizedGrid element */
+const contributorsGridRef = React.createRef();
+
```

Comment:
Having that in the global scope means a lack of isolation (which will cause issues if we try to instantiate the component multiple times) and potential memory leaks. I suggest moving this to the constructor, as `this.contributorsGridRef = ...`